content/en/real_user_monitoring/reactnative/_index.md

```diff
@@ -89,7 +89,16 @@ config.serviceName = 'com.example.reactnative';
 // Optional: let the SDK print internal logs above or equal to the provided level. Default is undefined (meaning no logs)
 config.verbosity = SdkVerbosity.WARN;
 
-await DdSdkReactNative.initialize(config);
+const config = new DatadogProviderConfiguration();
```

Comment:
I can't make the github suggestion from here, so let me describe the change. 
Instead of adding this line, you should change the line 70 from 

```
const config = new DdSdkReactNativeConfiguration(
```

to 

```
const config = new DatadogProviderConfiguration(
```

Then keep the rest :)